pkg/platform/abstract/platform.go

```diff
@@ -544,6 +590,10 @@ func (ap *Platform) getMessageAndArgs(message string, args *string, log []byte)
 	}
 
 	if additionalKwargs != nil {
+		if workerID != "" {
+			additionalKwargs["worker_id"] = workerID
```

Comment:
Are you enriching wid which you may have found from logger name into a kwarg?
Why? I think it's best to not mess with the log so we know that whatever was on it was there to begin with and not added in the gathering process